env.sh.example

```diff
@@ -26,5 +26,5 @@ export RESOURCE_SUFFIX=
 
 # Leave as-is
 export VERTEX_SA_EMAIL=vertex-pipelines@${VERTEX_PROJECT_ID}.iam.gserviceaccount.com
-export PIPELINE_FILES_GCS_PATH=gs://${VERTEX_PROJECT_ID}-pl-assets/${RESOURCE_SUFFIX}
+export PIPELINE_FILES_GCS_PATH=gs://${VERTEX_PROJECT_ID}-pl-assets
```

Comment:
Is this the source of issue? If `RESOURCE_SUFFIX` is empty, this will create `PIPELINE_FILES_GCS_PATH=gs://...-pl-assets/` leading to `gsutil -m rsync -r -d ./pipelines/assets "gs://...-pl-assets//assets"` ?

Reply:
if so, could always require `RESOURCE_SUFFIX` to start with `/`. then we don't need L18-26 in this PR.

Reply:
`RESOURCE_SUFFIX` is also used to name BigQuery tables. It would work, but also look weird. I'm easy though

Reply:
true, then ignore my suggestion